eric_sse/connection.py

```
@@ -29,6 +32,26 @@ def queue(self) -> Queue:
     def id(self) -> str:
         return self.__id
 
+    def send_message(self, msg: MessageContract):
+        try:
+            self.__queue.push(msg)
+        except Exception as e:
+            for handler in self.__queues_error_handlers:
+                handler.handle_push_error(msg=msg, exception=e)
+            raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)